src/common/clib-settings.h

```diff
@@ -9,6 +9,6 @@
 #define MAX_THREADS 12
 #endif
 
-const char *manifest_names[] = {"clib.json", "package.json", 0};
+static const char *manifest_names[] = {"clib.json", "package.json", 0};
```

Comment:
This will issue a compilation warning:

```
src/common/clib-settings.h:12:20: warning: ‘manifest_names’ defined but not used [-Wunused-variable]
 static const char *manifest_names[] = {"clib.json", "package.json", 0};
 ```
 It think the correct way to do it is to declare the variable as extern in the header file:
 ```
extern const char *manifest_names[];
```

And in for example a `clib-settings.c`:
```
const char *manifest_names[] = {"clib.json", "package.json", 0};
```
And of course `clib-settings.c` should be added to each binaries.

Reply:
Yes, you are right, that is better solution.